transportable-udfs-plugin/src/main/java/com/linkedin/transport/plugin/TransportPlugin.java

```diff
@@ -192,9 +196,33 @@ private SourceSet configureSourceSet(Project project, Platform platform, SourceS
           task.dependsOn(project.getTasks().named(inputSourceSet.getClassesTaskName()));
         });
 
-    project.getTasks()
-        .named(outputSourceSet.getCompileTaskName(platform.getLanguage().toString()))
-        .configure(task -> task.dependsOn(generateWrappersTask));
```

Comment:
Can we leave this as is and then replace the switch block below with `if (platform.getLanguage() = JAVA)`?

Reply:
updated